src/test/GeometryStore.test.ts

```diff
@@ -149,6 +149,9 @@ describe("GeometryStore module", () => {
         });
         test("if a request fails, returns a sphere with an error message", async () => {
             const store = new GeometryStore();
+            // TODO threejs' OBJLoader now uses the Request api under the hood, and Request is
+            // not available in nodejs (the environment our tests run in) without a polyfill
+            // such as node-fetch. I tried to add node-fetch here but I couldn't get it to work.
             const returned = await store.mapKeyToGeom(1, {
```

Comment:
Confused...  So `store.mapKeyToGeom()` uses threejs' OBJLoader which uses the Request API which doesn't work in nodejs? And using node-fetch wouldn't work? So how come the tests still pass?

Reply:
the tests pass because of the extra try-catch I added.  The error is "Request is not defined" which was throwing but not caught as part of the FileLoader's own error handling.